maquetado/setup/index.html

```diff
@@ -9,15 +9,100 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Front-End Tools</h1>
-  <div class="tech">
-    <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS" />
-    <h2 class="tech-title">ReactJS</h2>
-    <h4 class="tech-date">Release:2013</h4>
-    <p class="tech-text">
-      React is a declarative, component-based
-       JavaScript library for building user interfaces.
-     </p>
+  <div class="title-techs">
+    <h1 class="title">Front-End Tools</h1>
+    <div class= "techs-containers">
+      <div class="tech">
+        <img class="tech-logo" src="assets/react_icon.svg" alt="ReactJS" />
+        <h2 class="tech-title">ReactJS</h2>
+        <h4 class="tech-date">Release:2013</h4>
+        <p class="tech-text">
+          React is a declarative, component-based
+          JavaScript library for building user interfaces.
+         </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/angular_icon.svg" alt="AngularJS" />
+        <h2 class="tech-title">AngularJS</h2>
+        <h4 class="tech-date">Release:2010</h4>
+        <p class="tech-text">
+          AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript
+          framework for building SPAs with an MVC architecture.
+         </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/vue_icon.svg" alt="VueJS" />
+        <h2 class="tech-title">VueJS</h2>
+        <h4 class="tech-date">Release:2014</h4>
+        <p class="tech-text">
+          Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript
+          framework for building user interfaces.
+         </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/elm_icon.svg" alt="Elm" />
+        <h2 class="tech-title">Elm</h2>
+        <h4 class="tech-date">Release:2016</h4>
+        <p class="tech-text">
+          Elm is a domain-specific programming language for declaratively creating
+          web browser-based graphical user interfaces.
+         </p>
+      </div>
+      <div class="tech">
+        <img class="tech-logo" src="assets/polymer_icon.svg" alt="Polymer" />
+        <h2 class="tech-title">Polymer</h2>
+        <h4 class="tech-date">Release:2015</h4>
+        <p class="tech-text">
+          Polymer is an open-source JavaScript library for building web
+          applications using Web Components.
+         </p>
+      </div>
+    </div>
+  </div>
+  <div class="title-techs">
+    <h1 class="title">Back-End Tools</h1>
```

Comment:
Same, you could have more than one title in your page. Therefore, it's good practice to be specific with class naming. It could be `tech-section-title` or `tech-area-title`. 

This becomes easier when you start using css modules in react :+1: